api/database/factories/UserFactory.php

```diff
@@ -87,6 +89,7 @@ public function definition()
             ]),
             'is_gov_employee' => $this->faker->boolean(),
             'interested_in_later_or_secondment' => $this->faker->boolean(),
+            'department' => $randomDepartment ? $randomDepartment->id : null,
```

Comment:
for testing purposes, it might be handy to have department contingent on is_gov_employee so you don't end up with people with a department but are not gov employees